specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2021-05-01/managedClusters.json

```diff
@@ -4220,46 +4207,67 @@
       ],
       "x-ms-enum": {
         "name": "KubeletDiskType",
-        "modelAsString": true
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "OS",
+            "description": "Kubelet will use the OS disk for its data."
+          },
+          {
+            "value": "Temporary",
+            "description": "Kubelet will use the temporary disk for its data."
+          }
+        ]
       },
-      "description": "KubeletDiskType determines the placement of emptyDir volumes, container runtime data root, and Kubelet ephemeral storage. Allowed values: 'OS', 'Temporary' (preview)."
+      "description": "Determines the placement of emptyDir volumes, container runtime data root, and Kubelet ephemeral storage."
     },
     "KubeletConfig": {
```

Comment:
This type has a number of fields that could be documented as enums if we wanted (and if it's not a breaking change from what we have). Need to confirm if people want that.

Reply:
Spoke with Juan-Lee and he suggested that we can document these as enums because the service really does reject if you get it wrong

Reply:
Have done so